### DISCUSSION TITLE
Error: Directory provided does not exist

### DISCUSSION THREAD

**Discussion by 1Sage1** [satijalab/seurat#4880] [Q&A] [answered]
On running this line, I get the error:Directory provided does not exist
pbmc.data <- Read10X(data.dir = "../data/pbmc3k/filtered_gene_bc_matrices/hg19/")

Am I missing something? 
P.S. I am beginning to learn Seurat.

Thanks!

**Comment by timoast** [comment, marked as answer]
It looks like the directory path you provided does not exist on your computer. You need to provide the path to the file on your computer, rather than copying the exact path shown in the online vignettes.

**Reply by 1Sage1** [reply]
Thank you very much

**Reply by immuno345** [reply]
Where can we find the data to put in our directory? Also struggling with this step

**Reply by tfrudakis** [reply]
I also struggle with this step.  I had to re-type the entire directory path from C://Users/Admin all the way down.  Simply using "../filtered_gene_bc_matrices/hg19/" from the working directory containing that folder just did not work, so somehow I had it in absolute rather than relative path mode.  I still do not understand how though...

**Reply by shuzhe0426** [reply]
I struggle with this step as well. When I type  # Initialize the Seurat object with the raw (non-normalized data)
> pbmc <- CreateSeuratObject(counts = pbmc.data, project = "pbmc3k")
It shows Error: object 'pbmc.data' not found.
